Lob-API-public.yml

```diff
@@ -130,9 +130,6 @@ tags:
       in the future by supplying the `send_date` parameter, that will override any
       cancellation window you may have for that product.
 
-  - name: Certificates
-    description: A certificate store to manage client identities.
-
```

Comment:
Why was the certificates resource deleted? Is it because Verizon is the only one using it? Because by that logic, we should exclude batch endpoints as well, which I just finished the first draft for. But I don't see anything wrong with leaving it in, especially if it may pique the interest of other customers.

Reply:
@jho44 It's a private endpoint––I confirmed with PMAPI. Therefore, it absolutely shouldn't be in Lob OpenAPI. 

Batch should also not be in Lob OpenAPI if it's not a public endpoint (and that includes endpoints hidden by account feature flags, like `intl_autocompletions`––I had to remove that from Lob OpenAPI a while back). I mentioned creating a private repo for internal endpoints, Lob AdminAPI: _that_ is where things like `certificates` go.

Reply:
To confirm @sudoku-lord - the private endpoints will find a home in a new repo called lob-adminiapi